src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/search/crosswikisearch/CrossWikiSearchPageObject.java

```diff
@@ -185,7 +185,7 @@ public void verifyNoResultsCaption() {
   public void verifyThumbnails(int number) {
     Assertion.assertNumber(thumbnails.size(), number, "checking number of thumbnails");
     for (WebElement elem : thumbnails) {
-      Assertion.assertStringContains(elem.getAttribute("src"), ".png");
+      Assertion.assertStringContainsOneOfTwoElements(elem.getAttribute("src"), ".png", ".jpg");
```

Comment:
@ludwikkazmierczak - can we instead assert that `elem.getAttribute("src")` matches `\.(png|jpg)` regexp?